pkg/operator/ceph/pool/radosnamespace/controller.go

```diff
@@ -212,13 +223,27 @@ func (r *ReconcileCephBlockPoolRadosNamespace) reconcile(request reconcile.Reque
 	}
 	r.clusterInfo.Context = r.opManagerContext
 
+	cephRNSList := &cephv1.CephBlockPoolRadosNamespaceList{}
```

Comment:
Move this block inside the `if` block for checking if the rns is deleted. No need to list it for non-deleted CRs.

Reply:
done